cinn/optim/transform_gpu_forloop.cc

```diff
@@ -179,14 +179,16 @@ class ReplaceIndexToBindExpr : public ir::IRMutator<> {
     auto *schedule_block_realize = expr->As<ir::ScheduleBlockRealize>();
     CHECK(schedule_block_realize->schedule_block.As<ir::ScheduleBlock>());
     auto iter_values = schedule_block_realize->iter_values;
-    auto body_copy   = schedule_block_realize->schedule_block.As<ir::ScheduleBlock>()->body;
+    auto body        = schedule_block_realize->schedule_block.As<ir::ScheduleBlock>()->body;
```

Comment:
Use type deduction only to make the code clearer or safer, and do not use it merely to avoid the inconvenience of writing an explicit type.

These auto may be hard to read for new CINN developers.

Reference: https://google.github.io/styleguide/cppguide.html#Type_deduction